csharp/src/Microsoft.ML.OnnxRuntime/targets/netstandard/targets.xml

```diff
@@ -6,12 +6,12 @@
     the PlatformTarget is empty, and you don't know until runtime (i.e. which dotnet.exe)
     what processor architecture will be used.
   -->
-    <Error Condition="('$(PlatformTarget)' != 'x64' AND '$(PlatformTarget)' != 'x86' AND '$(PlatformTarget)' != 'AnyCPU') AND
+    <Error Condition="('$(PlatformTarget)' != 'x64' AND '$(PlatformTarget)' != 'arm64' '$(PlatformTarget)' != 'x86' AND '$(PlatformTarget)' != 'AnyCPU') AND
```

Comment:
arm64' '$(PlatformTarget)' => arm64' AND '$(PlatformTarget)'

Reply:
whoops. fixed.